### ISSUE TITLE
Статистика не отправляется

### ISSUE THREAD

**Issue by Xewus** [ScrapeOps/scrapeops-scrapy-sdk#4]
When it is written in the settings
`BASE_DIR = Path(__file__).resolve().parent.parent`
the extension does not send statistics to the API. 
Even if it is written in another module and then imported into the settings.

But this works OK if write
`BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))`

**Comment by ronaldokun** [comment]
Yes, I stumbled upon the same thing. If you use the pathlib module anywhere in the settings it's not sending statistics to the API.
